### ISSUE TITLE
Could we change the UI Grid style

### ISSUE THREAD

**Issue by Bitchy** [angular-ui/ui-grid#4016]
Could we change the the scrollbar of the UI Grid as below picture show? if Yes, how do change it, does  anyone konw how to modify the css file for it?
![qq 20150721164255](https://cloud.githubusercontent.com/assets/4770761/8796486/b3618c30-2fc7-11e5-95b2-3a15ca95cd64.png)

**Comment by c0bra** [comment]
UI Grid uses native scrolling so the scrollbars are dependent on the
browser and OS.

**Comment by Bitchy** [comment]
Yes, you are right.  We want to use a third party jQuery library to custom the scrollbar, reference here:http://manos.malihu.gr/jquery-custom-content-scroller/, but I am not sure it support the UI Grid, do you know about it?

**Comment by c0bra** [comment]
I haven't heard of anyone using it with UI Grid so I can't speculate unfortunately.

**Comment by c0bra** [comment]
If you give it a try and it works/doesn't work let me know.

**Comment by viktordavidovich** [comment]
Also have the same issue. I want to use custom scrollBar with ui-grid. Please somebody give me information about this. Thank you.

**Comment by Gonkaro** [comment]
Hello,

Same issue here.

When I try to use custom scrollbars they don't seem to work with the ui-grid.

I guess they are not compatible...

**Comment by amit1702** [comment]
I also tried it, not working for me also. here is the fiddle.

https://jsfiddle.net/amiku1702/hmay2ya8/7/

**Comment by deepakambastha28** [comment]
we can use third party angular scrollbar.

**Comment by mobichel** [comment]
Hi all,

here is example how to modify html & less to get styled scrollbars: #5951

**Comment by vinay5c8** [comment]
.ui-grid-viewport::-webkit-scrollbar-track
{
    -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
    border-radius: 10px;
    background-color: #F5F5F5;
}

.ui-grid-viewport::-webkit-scrollbar
{
    width: 12px;
    background-color: #F5F5F5;
}

.ui-grid-viewport::-webkit-scrollbar-thumb
{
    border-radius: 10px;
    -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,.3);
    background-color: #555;
}

use this

**Comment by mobichel** [comment]
@vinay5c8 thanks, 

it will work only for webkit based browsers. 

You can see how to get consistent scrollbars across browsers in ui-grid [here](https://github.com/mobichel/ui-grid/tree/styled-scrollbars).

**Comment by mobichel** [comment]
Here is example how to stlye scrollbar without modification in ui-grid: [example](https://jsfiddle.net/mobichel/qpnfL06n/4/)

**Comment by mportuga** [comment]
We should document the example provided
